TenancyInformationApi.Tests/V1/Gateways/TenancyGatewayTests.cs

```diff
@@ -56,14 +56,41 @@ public void GetByIdWillOnlyGetLookupValuesForZAGTypeLookups()
             UhContext.UhTenancyAgreementsType.Add(nonZagAgreementTypeLookup);
             UhContext.SaveChanges();
 
-            var uhTenancy = AddTenancyAgreementToDatabase(tagRef, agreementTypeLookup, tenureTypeLookup);
-
             var expected = uhTenancy.ToDomain(agreementTypeLookup);
             var response = _classUnderTest.GetById(uhTenancy.TenancyAgreementReference);
 
             response.Should().BeEquivalentTo(expected);
         }
 
+        [Test]
+        public void ListTenanciesWithNoSavedTenanciesReturnsAnEmptyList()
+        {
+            _classUnderTest.ListTenancies().Should().BeEmpty();
+        }
+
+        [Test]
+        public void ListTenanciesGivenNoQueryParametersWillReturnAllSavedTenancies()
+        {
+            var savedEntities = new List<(UhTenancyAgreement uhTenancy, UhTenureType tenureTypeLookup, UhAgreementType agreementTypeLookup)>
```

Comment:
Are you creating the list with the returned values from the function?

Reply:
Yeah, so a list of tuples.

Reply:
neat